.golangci.yml

```diff
@@ -62,7 +62,6 @@ linters:
     - bidichk             # checks for dangerous unicode character sequences
     - bodyclose           # checks whether HTTP response body is closed successfully
     - contextcheck        # check the function whether use a non-inherited context
-    - deadcode            # finds unused code
```

Comment:
does this need to be removed?

Reply:
golangci-lint gives a warning that the linter is deprecated and no longer updated by its owner. i included all of these linters because the Hugo plugin does as well. what are your thoughts on keeping it?

Reply:
also, after looking into the error again it seems like all three of these were replaced by `unused`